scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -344,6 +369,7 @@ normalized_sce <- scuttle::logNormCounts(qcfiltered_sce)
 
 This creates a new "assay" in the `normalized_sce` object, `logcounts`, which contains the normalized count values for each cell and gene.
 (The data here are not _actually_ the log of the counts, since we also applied the scaling factors, but that is what we have.)
```

Comment:
```suggestion
(The data here are not _actually_ the log of the counts, since we also applied the scaling factors, but that name is used for historicial reasons.)
```